pkgs/development/compilers/vlang/default.nix

```diff
@@ -2,29 +2,29 @@
 
 stdenv.mkDerivation rec {
   pname = "vlang";
-  version = "weekly.2022.20";
+  version = "weekly.2022.43";
```

Comment:
2 days ago, 0.3.2 is released which is newer than this. Shouldn't we just use that one?